charts/opentelemetry-collector/values.schema.json

```diff
@@ -27,6 +27,29 @@
         ""
       ]
     },
+    "presets": {
+      "type": "object",
+      "properties": {
+        "logsCollection": {
+          "type": "object",
+          "properties": {
+            "enabled": {
```

Comment:
Let's add `"additionalProperties": false` in all the objects. Also schema for `includeCollectorLogs` is missed

Reply:
Good catch